Rlog/View/WorkSpace/WorkSpaceCreateConfirmationView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  WorkSpaceCreateConfirmationView.swift
+//  Rlog
+//
+//  Created by 송시원 on 2022/10/19.
+//
+
+// TODO: 네비게이션 연결, 뷰모델 연결, 고민 공유, 컨포넌트 적용
+
+import SwiftUI
+
+struct WorkSpaceCreateConfirmationView: View {
+    var body: some View {
+        // 고민: 폭이 한계까지 닿지 않아서 가운데정렬이 되는데, 컨포넌트를 가져오면 해결된다. 이럴 경우 다른 방식으로 미뤄두는게 필요할까?
```

Comment:
```suggestion
        // TODO: 고민: 폭이 한계까지 닿지 않아서 가운데정렬이 되는데, 컨포넌트를 가져오면 해결된다. 이럴 경우 다른 방식으로 미뤄두는게 필요할까?
```

TODO 로 보통의 주석이 아닌 표시가 필요해보여요 ~ !

Reply:
오 투두 적극 활용하겠습니다!
감사합니다~!